front/components/assistant/conversation/actions/VisualizationActionIframe.tsx

```diff
@@ -167,6 +182,46 @@ export function VisualizationActionIframe({
     [codeFullyGenerated, iframeLoaded, isErrored]
   );
 
+  const handleVisualizationRetry = async () => {
```

Comment:
We now keep those api calls in hook as well, example: 

https://github.com/dust-tt/dust/blob/1ba28be6d1d3d44e6944d0011570cfe18df4df95/front/lib/swr/vaults.ts#L233

Reply:
Ok -- I think we should rename the folder into `api_hooks` or something instead of `swr`